client/src/styles/breakpoints.scss

```diff
@@ -0,0 +1,57 @@
+// Device breakpoint variables
+$breakpoint-xs: 320px;
+$breakpoint-sm: 576px;
+$breakpoint-md: 768px;
+$breakpoint-lg: 992px;
+$breakpoint-xl: 1200px;
+$breakpoint-xxl: 1400px;
```

Comment:
there are breakpoints in the bootstrap styles, do we need different?

Reply:
example
```
@include media-breakpoint-down(lg) {
```

Reply:
+